base/src/main/java/org/aya/ref/DefVar.java

```diff
@@ -61,6 +61,10 @@ private DefVar(Concrete concrete, Core core, @NotNull String name) {
     return new DefVar<>(null, null, name);
   }
 
+  public static OpDecl resolveOpDecl(@NotNull DefVar<?,?> defVar, @NotNull ImmutableSeq<String> moduleName){
+    return defVar.opDeclRename.getOrDefault(moduleName, defVar.opDecl);
+  }
+
```

Comment:
It is not needed to be static. And it would be nice if `@Nullable OpDecl`

Reply:
resolved